requirements/.python-version

```diff
@@ -1 +1 @@
-3.6.9
+3.9.5
```

Comment:
I'm not sure what the purpose of this file is

Reply:
Ah, dependabot: https://github.com/dimagi/commcare-hq/commit/8aaa93d6dbef37c332dd1c8eb70d9f74c026ae86